chalicelib/checks/wfr_checks.py

```diff
@@ -1,1448 +1,1456 @@
-        head_info = my_s3_util.does_key_exist(a_file['upload_key'], my_bucket)
+        file_id = a_file["accession"]
+        head_info = my_s3_util.does_key_exist(a_file["upload_key"], my_bucket)
         if not head_info:
             no_s3_file.append(file_id)
             continue
         md5_report = wfr_utils.get_wfr_out(a_file, "md5", key=my_auth, md_qc=True)
-        if md5_report['status'] == 'running':
+        if md5_report["status"] == "running":
             running.append(file_id)
-        elif md5_report['status'].startswith("no complete run, too many"):
+        elif md5_report["status"].startswith("no complete run, too many"):
             problems.append(file_id)
-        # Most probably the trigger did not work, and we run it manually
-        elif md5_report['status'] != 'complete':
+        # most probably the trigger did not work, and we run it manually
+        elif md5_report["status"] != "complete":
             missing_md5.append(file_id)
-        # There is a successful run, but status is not switched, happens when a file is reuploaded.
-        elif md5_report['status'] == 'complete':
+        # there is a successful run, but status is not switched, happens when a file is reuploaded.
+        elif md5_report["status"] == "complete":
             not_switched_status.append(file_id)
     if no_s3_file:
-        check.summary = 'Some files are pending upload'
-        msg = str(len(no_s3_file)) + '(uploading/upload failed) files waiting for upload'
+        check.summary = "Some files are pending upload"
+        msg = (
+            str(len(no_s3_file)) + "(uploading/upload failed) files waiting for upload"
+        )
         check.brief_output.append(msg)
-        check.full_output['files_pending_upload'] = no_s3_file
+        check.full_output["files_pending_upload"] = no_s3_file
     if running:
-        check.summary = 'Some files are running md5runCGAP'
-        msg = str(len(running)) + ' files are still running md5runCGAP.'
+        check.summary = "Some files are running md5runCGAP"
+        msg = str(len(running)) + " files are still running md5runCGAP."
         check.brief_output.append(msg)
-        check.full_output['files_running_md5'] = running
+        check.full_output["files_running_md5"] = running
     if problems:
-        check.summary = 'Some files have problems'
-        msg = str(len(problems)) + ' file(s) have problems.'
+        check.summary = "Some files have problems"
+        msg = str(len(problems)) + " file(s) have problems."
         check.brief_output.append(msg)
-        check.full_output['problems'] = problems
-        check.status = 'WARN'
+        check.full_output["problems"] = problems
+        check.status = "WARN"
     if missing_md5:
-        check.allow_action = True
-        check.summary = 'Some files are missing md5 runs'
-        msg = str(len(missing_md5)) + ' file(s) lack a successful md5 run'
+        check.summary = "Some files are missing md5 runs"
```

Comment:
I'm noticing in this check that if multiple of these if statements are true, the check.summary gets overwritten. We should consider whether we are happy with the overwriting, or if we want a more general check summary such as "Issues found regarding md5 runs"